test/clt-tests/migration-es-ms/test-migration-es-ms.rec

```diff
@@ -0,0 +1,161 @@
+––– block: ../mysqldump/versions/init –––
```

Comment:
Wrong path. If this init.recb file is now used in both the mysqldump test and here it should be placed to a more neutral place, more over there's nothing about mysqldump/elasticdump in it.

Reply:
Done